apps/ui/src/components/EditorVotingType.vue

```diff
@@ -34,11 +44,17 @@ function handleVoteTypeSelected(type: VoteType) {
     <h4 class="eyebrow mb-2.5">Voting type</h4>
     <button
       type="button"
-      class="border rounded-xl py-2.5 px-3 flex gap-3 text-left relative border-skin-content w-full"
-      @click="modalOpen = true"
+      class="border rounded-xl py-2.5 px-3 flex text-left relative border-skin-content w-full"
```

Comment:
Let's disable this button this way it's not focusable and more accessible. Then we don't need to check `hasMultipleVotingType` on handler.